src/loader/EventLoader.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import DataLoader from 'dataloader';
+import { Event as EventModel } from '../model';
+import { connectionFromMongoCursor, mongooseLoader } from '@entria/graphql-mongoose-loader';
+
+import type { ConnectionArguments } from 'graphql-relay';
+import type { GraphQLContext } from '../TypeDefinition';
+
+type Schedule = {
+  talker: string,
+  title: string,
+  description: string,
+  time: number,
+};
+
+export type EventType = {
+  id: string,
+  _id: string,
```

Comment:
this is a ObjectID from 'mongoose'

Reply:
i think we dont have this type i think is fine as a string